### DISCUSSION TITLE
fabric-api provides incorrect mapping (?)

### DISCUSSION THREAD

**Discussion by SirEdvin** [cc-tweaked/CC-Tweaked#1426] [Q&A] [answered]
### Minecraft Version

1.19.x

### Version

1.104.0

### Details

So, I am working on port of Peripheralium to 1.19.4 and to multi-loader structure. 

I am not good at gradle, but I have semes implemented something, like here: https://github.com/SirEdvin/Peripheralium/blob/1.19.4/projects/fabric/build.gradle.kts

I have tried to use fabric-api as dependency, but received a lot strange errors with mapping 
<img width="622" alt="Screenshot 2023-04-29 at 21 39 22" src="https://user-images.githubusercontent.com/3713151/235319188-e4d9490d-2cf1-442d-934e-d307b92d1a8f.png">

However, if I replace fabric-api with fabric, everything processing just fine.

**Comment by SquidDev** [comment, marked as answer]
You need to update to Loom 1.1. Older versions of Loom don't pick up that the API is a mod (given it's not really), and so needs to be remapped.

**Reply by SirEdvin** [reply]
Thanks, works now :)
